RamanData/300/.~lock.R_300_2.csv#

```diff
@@ -0,0 +1 @@
+,xiangyu,CR7,15.02.2018 10:17,file:///home/xiangyu/.config/libreoffice/4;
```

Comment:
Delete this file as well.

Reply:
I deleted all .~lock files using a for loop in Bash.